src/main/scala/io/ino/solrs/SolrServers.scala

```diff
@@ -119,6 +119,8 @@ class CloudSolrServers[F[_]](zkHost: String,
 
   @volatile
   private var collectionToServers = Map.empty[String, IndexedSeq[SolrServer]].withDefaultValue(IndexedSeq.empty)
+  @volatile
+  private var aliasToCollections = Map.empty[String, IndexedSeq[String]].withDefaultValue(IndexedSeq.empty)
```

Comment:
Is `withDefaultValue(IndexedSeq.empty)` really needed? AFAICS it's used nowhere.